### PR TITLE
[WIP]scheduler: add command priority

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
High priority command will ignore write flow control.
@siddontang @disksing @javaforfun PTAL

### CODE REVIEW

**Comment by disksing** [comment]
I think we need to reorder pending commands and process commands with higher priority first.

**Comment by zhangjinpeng1987** [comment]
@disksing For what purpose we should reorder the command?

**Comment by javaforfun** [comment]
LGTM

**Comment by disksing** [comment]
To make import commands get executed as fast as possible? You only ignore the context limit for these commands, may lead to the accumulation of commands.

File: src/storage/mod.rs [hunk 1]
```diff
@@ -82,6 +82,13 @@ pub enum StorageCb {
     Locks(Callback<Vec<LockInfo>>),
 }
 
+#[derive(PartialEq, Clone, Copy)]
+pub enum CmdPri {
+    Low,
```
Comment (siddontang) [inline src/storage/mod.rs]: seem we don't have Low command now.

File: src/storage/mod.rs [hunk 2]
```diff
@@ -239,13 +232,10 @@ impl Command {
...
 
-    pub fn priority(&self) -> CmdPri {
+    pub fn need_flow_control(&self) -> bool {
         match *self {
-            Command::Commit { .. } |
-            Command::Rollback { .. } |
-            Command::Cleanup { .. } |
-            Command::ResolveLock { .. } => CmdPri::High,
-            _ => CmdPri::Normal,
+            Command::Prewrite { .. } => true,
```
Comment (siddontang) [inline src/storage/mod.rs]: How about 1PC, no need flow control?

**Comment by ngaut** [comment]
LGTM

**Pull request closed without merging**